update

```diff
@@ -4,17 +4,33 @@
 
 const fs = require('fs')
 const path = require('path')
+const semver = require('semver')
 
 // This script updates the index files list.js and list.txt in the bin directory,
 // as well as the soljson-latest.js files.
 
 fs.readdir(path.join(__dirname, '/bin'), function (err, files) {
   if (err) return err
 
+  function buildVersion (build) {
```

Comment:
Actually if we rebuild it with MM.DD we don't even need semver to do the comparison.